src/pages/ServerRoleSummary/DiskModelDetails.js

```diff
@@ -251,31 +362,61 @@ class DiskModelDetails extends Component {
   }
 
   checkSecondDetailsDataToSave = () => {
-    if (this.state.secondDetails === 'vg') {
-      return (this.state.volumeGroup.name && this.state.volumeGroup.name !== '') &&
-        (this.state.physicalVolumes.length > 0) && (this.state.logicalVolumes.length > 0);
+    if (this.secondDetails === 'vg') {
+      if (this.vgMode === MODE.ADD) {
+        return this.state.volumeGroup.name && this.state.volumeGroup.name !== '' &&
+          this.state.physicalVolumes.length > 0 && this.state.logicalVolumes.length > 0;
+      } else {
+        const pvs = this.state.physicalVolumes.slice().sort();
+        return (this.state.volumeGroup.name && this.state.volumeGroup.name !== '' &&
```

Comment:
some of these if statements are a bit tough to track, might help to have a comment explaining the gist of whats going on